website/src/pages/Compare/Compare.jsx

```diff
@@ -0,0 +1,272 @@
+                    const responseMacrobench = await fetch(`${import.meta.env.VITE_API_URL}macrobench/compare?rtag=${gitRefRight}&ltag=${gitRefLeft}`)
+
+                    const jsonDataMacrobench = await responseMacrobench.json();
+                    
+                    setDataMacrobench(jsonDataMacrobench)
+                    setIsLoading(false)
+                } catch (error) {
+                    console.log('Error while retrieving data from the API', error);
+                    setError(errorApi);
+                    setIsLoading(false)
+                }
+            };
+            fetchData();
+        
+    }, [gitRefLeft, gitRefRight])
+
+    useEffect(() => {
+            const fetchData = async () => {
+                try {
+                    const responseMicrobench = await fetch(`${import.meta.env.VITE_API_URL}microbench/compare?rtag=${gitRefRight}&ltag=${gitRefLeft}`)
+                    
+                    const jsonDataMicrobench = await responseMicrobench.json();
+                    console.log(jsonDataMicrobench)
+                    setDataMicrobench(jsonDataMicrobench)
+                } catch (error) {
+                    console.log('Error while retrieving data from the API', error);
+                    setError(errorApi);
+                }
+            };
+            fetchData();
+         
+    }, [gitRefLeft, gitRefRight])
+
+    // Changing the URL relative to the reference of a selected benchmark.
+    // Storing the carousel position as a URL parameter.
+    const navigate = useNavigate();
+    
+    useEffect(() => {
+        navigate(`?ltag=${gitRefLeft}&rtag=${gitRefRight}&ptag=${currentSlideIndex}&ptagM=${currentSlideIndexMobile}`)
+    }, [gitRefLeft, gitRefRight, currentSlideIndex, currentSlideIndexMobile]) 
+    
+    const handleInputChangeLeft = (e) => {
+        setGitRefLeft(e.target.value);
+      };
+    
+
+      const handleInputChangeRight = (e) => {
+        setGitRefRight(e.target.value);
+      };
+
+      const handleSlideChange = (swiper) => {
+        setCurrentSlideIndex(swiper.realIndex);
+    };
+
+    const slicedRef = gitRefLeft.slice(0, 8);
+    return (
+        <div className='compare'>
+            <div className='compare__top justify--content'>
+                <div className='compare__top__text'>
+                    <h2>Compare</h2>
+                    <span>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer a augue mi.
+                            Etiam sed imperdiet ligula, vel elementum velit.
+                            Phasellus sodales felis eu condimentum convallis.
+                            Suspendisse sodales malesuada iaculis. Mauris molestie placerat ex non malesuada.
+                            Curabitur eget sagittis eros. Aliquam aliquam sem non tincidunt volutpat. 
+                    </span>
+                    
+                </div>
+                <figure className='compareStats'></figure>
```

Comment:
I think we can get rid of that small header section on both search and comparison.